xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVAAPIGL.cpp

```diff
@@ -185,14 +212,15 @@ bool CRendererVAAPI::UploadTexture(int index)
     return UploadNV12Texture(index);
   }
 
-  m_vaapiTextures[index].Map(pic);
-  m_buffers[index].m_srcTextureBits = m_vaapiTextures[index].GetBits();
+  m_buffers[index].m_srcTextureBits = m_vaapiTextures[index]->GetBits();
```

Comment:
this is wrong. why did you change the order? bits is set in Map()

Reply:
I vaguely remember this being introduced in a rebase, so probably rebase screwup